assets/airports/rjss.json

```diff
@@ -0,0 +1,964 @@
+        "V33" : ["MIS", "HPE", "SDE"],
+        "V34" : ["MQE", "MORIO", "HPE", "HANKA", "MAMRO", "GTC"],
+        "Y102": ["CAVES", "HPE", "METEL", "SDE", "HERON", "FKE"],
+        "Y106": ["MQE", "SDE", "RUBIS"],
+        "Y12" : ["CHE", "HWE", "TAPPI", "ARIKA", "OTOME", "GTC", "FINGA", "MBE", "TOLUM", "NAKTU", "KOHWA", "SHIMA", "AYANO", "RAYJO", "KEC", "LAXEL"],
+        "Y122": ["HWE", "OBAKO", "YOHKO", "ATUMI", "BUDOH", "INAHO", "GTC"],
+        "Y124": ["MQE", "HPE", "TOLEK", "HANKA", "MAMRO", "GTC"],
+        "Y142": ["GTC", "NESKO", "SAMON"],
+        "Y15" : ["TENRU", "MYOGI", "TADMI", "DANDY", "HEIKE", "INAWA", "SHIRO", "NIRAX", "OWLET"],
+        "Y303": ["GTC", "VIKET", "AKTIL", "SASAP", "ESKAS", "ENTAK"],
+        "Y312": ["KCC", "AZUMI", "PAPNA", "NAEBA", "MAGNA", "GTC", "KENSI", "YAYOI", "UWE"],
+        "Y347": ["IGROD", "EKVIK", "HANAB", "GOLDO", "GTC"],
+        "Y45":  ["GTC", "TERAD", "HAKBA", "GORYU", "IKUJI", "URUSI", "IMIZU", "OYABE", "KMC", "SONBU", "MADUL", "SAKYU", "MIHOU", "KYOKA", "CARPS", "MARCO", "SQRUM", "YANAI", "OOITA", "SOBOH", "RAFUL", "MOCOS", "HKC", "ISKID", "MAKRA", "SAZMA", "BOMAP", "ANOXA", "MEKAX", "HACHA", "ONC", "LEXUS", "OKUMA", "NHC"],
+        "Y512": ["ADNAP", "OATIS", "BEKEN", "SDE", "ELDAK", "GTC"],
+        "Y514": ["ASTER", "SDE"],
+        "Y515": ["KMC", "ZUWAI", "ETIGO", "VIKET", "AIZUH", "LANCE"],
+        "Y882": ["RIKYU", "ONUMA", "SANOH", "GYODA"],
+        "Z119": ["GTC", "YANAT", "FKE"]
+    },
+    "sids": {
+        "CUBIC3": {
+            "icao": "CUBIC3",
+            "name": "Cubic Three",
+            "rwy": {
+                "RJSS09": [["^_RWY2702DME", "A5+"], "^SS901"],
+                "RJSS27": [["^_RWY0902DME", "A5+"], "BUBLE"]
+            },
+            "body": [["CUBIC", "A150+"]],
+            "exitPoints": {
+                "SAMBO": ["RIDER", "SAITI", "SAMBO"]
+            },
+            "draw": [
+                ["SS901", "CUBIC"],
+                ["BUBLE", "CUBIC"],
+                ["CUBIC", "RIDER", "SAITI", "SAMBO*"]
+            ]
+        },
+        "DERBY3": {
+            "icao": "DERBY3",
+            "name": "Derby Three",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "^SS901", "ANEMO", "EBOSI"],
+                "RJSS27": [["_RWY0901DME", "A5+"], "^SS701", ["_SS701233001", "A13+"], "EBOSI"]
+            },
+            "body": [["DERBY", "A100+"]],
+            "exitPoints": {
+                "GTC": ["GTC"]
+            },
+            "draw": [
+                ["SS901", "ANEMO", "EBOSI"],
+                ["SS701", "EBOSI"],
+                ["EBOSI", "DERBY", "GTC*"]
+            ]
+        },
+        "IXE8": {
+            "icao": "IXE8",
+            "name": "Iwaki Eight",
+            "rwy": {
+                "RJSS09": [["_RWY2702DME", "A5+"], "^_RWY2703DME", "_SDE120005"],
+                "RJSS12": ["_SDE120005"],
+                "RJSS27": ["^_SDE120005"],
+                "RJSS30": ["_SDE120005"]
```

Comment:
The fixing of the coordinates has caused another issue which can be seen below. IXE8 and SDE6R departures off RWY30 should make a *left* turn to heading 090 to intercept the SDE 120 radial outbound. Due to the new coordinates however, they're making a right turn. This can be fixed by adding an invisible point on a 270 bearing from the SDE120 intercept point (`_SDE120005`), somewhere near the circle I've drawn below. This should also be added to RWY27, although the problem isn't as noticeable there. This fix will also work for the SDE6R departure.
![IXE8_2](https://user-images.githubusercontent.com/13125311/86649602-d0cb1500-bfd9-11ea-88de-c181824a57e6.PNG)